app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
```

Comment:
You shouldn't need to include this HTML boilerplate in your view templates, it should be handled by `layouts/application.html.erb`.